app/controllers/letters_controller.rb

```diff
@@ -43,6 +43,14 @@ def like
     end
   end
 
+  def destroy
+    @letter = current_user.letters.find(params[:id])
+    @letter.destroy
```

Comment:
no need of using instance variables as those are not being passed to view. Secondly, please use `destroy!` as the output of destroy is not being checked.

Reply:
done